src/components/20-molecules/dropdown/react/story.jsx

```diff
@@ -16,9 +16,9 @@ storiesOf('Components|Molecules/Dropdown/React', module)
   .add('Dropdown as uncontrolled React Component', () => {
     const div = document.createElement('div');
 
-    const item1 = text('First Item', 'Item A');
-    const item2 = text('Second Item', 'Item B');
-    const item3 = text('Third Item', 'Item C');
+    const item1 = text('First Item', '< CHF 1,000');
+    const item2 = text('Second Item', 'From CHF 1,000 to 10,000');
```

Comment:
"From CHF 1,000 to CHF 10,000"

Reply:
Is the same on axa.ch.

![Screenshot 2020-01-31 at 15 27 45](https://user-images.githubusercontent.com/38284563/73546899-471d8700-443e-11ea-8657-49a58f590877.png)